frontend/src/components/LeftMenu/LeftMenu.jsx

```
@@ -21,48 +24,48 @@ const menuItems = [
   { text: 'GET FEEDBACK', title: true },
   { text: 'Feedback', icon: <ChatIcon /> },
   { text: 'Surveys', icon: <ListIcon /> },
-];
+  { text: 'Support', icon: <SportsIcon />, route: 'https://github.com/bluewave-labs/bluewave-onboarding' }];
 
 function LeftMenu() {
   const navigate = useNavigate();
 
   const handleNavigation = (route) => {
-    if (route) navigate(route);
+    if (route && route.startsWith('/')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)